.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
```

Comment:
Last line doesn't end in newline :)

Reply:
I just looked at the repo, and we only use `*.(py|tpl|stf|txt|md)` - if we set those to `text`, that should do it?

Reply:
git might actually understand that syntax, but we should probably make 5 lines, each with a simple glob :)

Reply:
Ok added specific lines